packages/styles/scss/components/content-group/_content-group.scss

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../globals/imports';
+
+@mixin themed-items {
+  .#{$prefix}--content-group__title {
+    color: $text-01;
+  }
+}
+
+@mixin content-group {
+  .#{$prefix}--content-group {
+    @include carbon--make-col-ready;
+
+    @include carbon--breakpoint(sm) {
+      padding-top: $carbon--layout-03;
+      padding-bottom: $carbon--layout-05;
+
+      &__title {
+        margin-bottom: $carbon--layout-05;
+      }
+
+      &__container {
+        @include carbon--make-col(4, 4);
+      }
+    }
```

Comment:
I don't think there is a `sm` breakpoint in the code. I think you could remove the `@include carbon--breakpoint(sm) {}` and move all of it's content up a level and it would do the same thing.